markdown/the_photomode_wishlist.md

```diff
@@ -104,19 +106,21 @@ Since the game is paused in photomode and players don't need to engage with it i
 
 ### Post-processing
 
-Besides the effects we mentioned above that we would like to get rid of (or at least have control over), there are also other post-processing effects that we might not want to delete completely, such as bloom. Having control over engine-specific post-processing settings is also useful if the resources allow them.
+Besides the effects we mentioned above that we would like to get rid of (or at least have control over), there are also other post-processing effects that we might not want to delete completely, such as bloom or color correction. Having control over engine-specific post-processing settings is also useful if the resources allow them.
 
 ### Aspect ratio options
 
-Players may not be familiar with the concept of aspect ratios, nor how a 16:9 screen is not a common ratio in photography, and are therefore restricted by what games have to offer. Other aspect ratios such as 3:4 or 2:1 can be incredibly beneficial to what they are trying to achieve. Ideally these could be achieved by rendering the final image in the desired ratio, rather than with black bar overlays.
+Players may not be familiar with the concept of aspect ratios, nor how a 16:9 screen is not a common ratio in photography, and are therefore restricted by what games have to offer. Other aspect ratios such as 3:4 or 2:1 can be incredibly beneficial to what they are trying to achieve.
+
+Ideally, these could be achieved by rendering the final image in the desired ratio, rather than with black bar overlays like [how we do it on PC](basics.htm#composing-for-custom-aspect-ratios). But understanding the limitations consoles have, black bar overlays might still be useful.
```

Comment:
```suggestion
Ideally, these could be achieved by rendering the final image in the desired ratio — like [how it's done on PC](basics.htm#composing-for-custom-aspect-ratios) — rather than with black bar overlays. But understanding the limitations consoles have, black bar overlays might still be useful.
```

Reworded a bit to reduce confusion about what the PC method is.